README.rst

```diff
@@ -74,6 +74,11 @@ With content already, it will be aligned to the opening parenthesis::
 
 Existing indentation (including ``0``) in multiline strings will be kept, so this setting only applies to the indentation of new/empty lines.
 
+g:python_pep8_setlocal_indent
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, this plugin sets ``shiftwidth``, ``tabstop``, and ``softtabstop`` to 4 on startup. This behavior can be disabled adding ``let g:python_pep8_setlocal_indent = 0`` to your vimrc.
+
```

Comment:
Yes.  `0` would still disable it (in case you want to set it yourself), but you could specify 2 here then for example.